scripts/utils/turbo_boost.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```

Comment:
The path to this script seems wrong. In the loader repo, the address is `scripts/turbo_boost.sh`. What should we do in the end?

Reply:
Ignore this comment. I have just seen that you changed the path on the new PR to my PR.